.storybook/main.js

```diff
@@ -37,6 +37,9 @@ module.exports = {
     config.resolve.alias['webextension-polyfill'] = require.resolve(
       '../ui/__mocks__/webextension-polyfill.js',
     );
+    config.resolve.alias['@metamask/snaps-controllers'] = require.resolve(
```

Comment:
Why is this needed?

Reply:
this was done to fix the incorrect import, it looks like the latest dependencies fixes this. i'll remove it.